test/main.sh

```diff
@@ -110,6 +110,11 @@ cleanup() {
     if [ "${expandDmesg}" = "no" ]; then
       echo "::endgroup::"
     fi
+
+    echo "::group::crashes"
+    ls -la /var/crash
+    cat /var/crash/*
```

Comment:
TIL: I thought that apport crash reports were binary files :)

Looking at https://askubuntu.com/questions/1210651/how-to-read-or-open-crash-file-from-var-crash, I see that you can get away with just looking at the textual representation or use a tool to unpack them. If we need more than `cat`, I think we should consider attaching them as CI artifacts.